manuscripts/Dockerfile

```diff
@@ -11,12 +11,12 @@ CMD ["R", "-e", "rmarkdown::render('manuscript.Rmd')"]
 ## Non-interactive use:
 ## 
 ##     docker run --name rnexml cboettig/rnexml
-##     docker cp rnexml:/host/manuscript.pdf .
+##     docker cp rnexml:/home/rstudio/rnexml/manuscript.pdf `pwd`
 
 ## Equivalent to:
 ##  
-##     docker run --name rnexml --volumes $(pwd):/host \
-##       --workdir /host --user docker rocker/ropensci \
+##     docker run --name rnexml --volumes $(pwd):/home/rstudio/rnexml \
+##       --workdir /home/rstudio/rnexml --user docker rocker/ropensci \
```

Comment:
Understood re: `rocker/ropensci` versus `cboettig/rnexml`. I changed to `cboettig/rnexml` because `rocker/ropensci` would start to download hundreds of MBs of new layers after the `cboettig/rnexml` with all layers was already on my machine, and I didn't want to abuse the bandwidth of the coffeeshop :smile: . In the end I think the priority here should be on reproducibility and transparency, not on ultimate user convenience (I wouldn't expect so many people to rerun the manuscript that it's worth saving them some typing of copy&paste effort). It seems to me that reproducibility and transparency would best be served by the rocker/ropensci container, and spelling out the command line.